.config/jest/mocks/react-inlinesvg.tsx

```diff
@@ -0,0 +1,25 @@
+// Due to the grafana/ui Icon component making fetch requests to
```

Comment:
Add pepperdata copywright here and in all tsx files

Reply:
Most of these files were generated by the grafana CLI tool, should we still add the copyright to all files even though we didn't write this code?